src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -37,10 +37,10 @@ export class AddContentComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.newContentForm.valueChanges.subscribe((change: {newTitle: string, existingTitle: string}) => {
-      if (change.newTitle.length < 3 && change.existingTitle.length < 3) this.state = 'closed';
+      if (change.newTitle.length < 3 && change.existingTitle.length < 3) this.expanded = false;
       else {
         this.currentValue = change.newTitle.length > 2 ? change.newTitle : change.existingTitle;
```

Comment:
use `>=3`, it will be easier to fix if this number changes (may be a constant btw)

Reply:
Done